azure/batch/scripts/train_model_finetune_on_catalog.py

```diff
@@ -61,7 +71,7 @@
         kade_catalog['file_loc'].iloc[len(kade_catalog.index) - 1]))
 
     datamodule = GalaxyDataModule(
-        label_cols=cosmic_dawn_ortho_schema.label_cols,
+        label_cols=schema_dict[args.schema].label_cols,
```

Comment:
Should we take care of the case when the user passes in something not in schema_dict?

Reply:
I agree, i'll add this check

Reply:
I added a fallback for this